paper.tex

```diff
@@ -81,9 +81,53 @@ \section{Method}
 We used the aforementioned models for generating and detecting texts and images. As for image processors, one was already given in the description of the "InformatiCup2024" \cite{InformatiCup}, the image noising using the Gaussian noise. Image noise is the random variation of brightness or color information in images produced by the sensor and circuitry of a scanner or digital camera. The reason why the Gaussian one, also called normal noise, is being used, is because it is being caused by natural sources and the most common noise that affects images naturally \cite{mohammed2016study}. \\
 There are several types of noise that can affect images. Some of these noise models are: Gaussian noise, White noise, Fractal noise, Salt \& Pepper noise, Periodic noise, Quantization noise, Speckle noise, Poisson noise, Poisson-Gaussian noise, Structured noise, Gamma noise and Rayleigh noise \cite{gonzalez2009digital}. The three common types of image noise are: Gaussian noise, Salt \& Pepper noise, and Speckle noise \cite{al2010comparative}. That is why we used those three, as well as the Poisson noise.
 
-\section{Results}
+\section{Results + Discussion}
+In the following we will have a look at the results of the aforementioned experiment.
 
-\section{Discussion}
+
+In fig. \ref{stable_diffusion} can be seen how the different processors and their combinations effect an image generated by the stable diffusion generator. The bars are shown in packs of three where the most left one represents the nahrawy evaluator, the middle one is resnet18 and lastly on the right is the umm-maybe evaluators respectively. The different colors represent different output. Black simbolizes that the image before processing was detected as AI-generated and after processing as well. Grey shows that both are detected to be human made, whereas for red the process did the exact opposite of what it should do and fool the detector in a way that after processing it is seen as AI-generated whereas before as human made. Lastly the green bar does exactly what is desired. \\
+It can be seen in fig. \ref{fig:stable_diffusion}, that only the nahrawy evaluator has a little bit of red in it but also it is the only one with the output of 'human-human'. Compared to the dall-E generator in fig. \ref{fig:dallE} the amount of green is relatively low. There are also other differences as here only umm-maybe achieved 'human-ai' but also 'human-human'. 
+It is visible that the gaussian noise image processor as well as the salt-and-pepper one are the best processors. This can be seen as they alone achieve a great result as well as their combinations with other ones. It is also worth mentioning that these result are only achieved when using the nahrawy evaluator. Whereas resnet18 which can be seen in both figures has only the output of 'ai-ai'.
```

Comment:
```suggestion
It is visible that the gaussian noise image processor as well as the salt-and-pepper one are the best processors. This can be seen as they solely achieve a great result as well as their combinations with other ones. It is also worth mentioning that these results are only achieved when using the nahrawy evaluator. Whereas resnet18, which can be seen in both figures, couldn't be tricked by any processor.
```